marklogic-data-hub-central/ui/src/assets/mock-data/common.data.ts

```diff
@@ -142,8 +142,14 @@ const xmlSourceData = [
         { rowKey: 6, key: 'LastName', val: 'Smith' }
       ]
       },
-      { rowKey: 7, key: 'proteinCat', val: 'commercial' },
-      { rowKey: 8, key: 'proteinDog', val: 'retriever, golden, labrador', array: true}
+      {
+        rowKey: 7, key: 'nutFree:name', children: [
```

Comment:
Can you try using other existing mock data that end with `MultipleSiblings` than modifying this one.